File: BOJ/완전탐색/2563.py
# 2563 색종이
# https://www.acmicpc.net/problem/2563
import sys

input = sys.stdin.readline

board = [[0] * 100 for _ in range(100)]
answer = 0

N = int(input())
for _ in range(N):
    x, y = map(int,input().split())

    for i in range(y+10):
        for j in range(x+10):
            board[i][j] = 1

for i in range(100):
    for j in range(100):
        if board[i][j]:
            answer += 1

print(answer)

File: BOJ/완전탐색/2563.cpp
// 2563 색종이
// https://www.acmicpc.net/problem/2563
#include <iostream>

using namespace std;

int board[101][101];

int main(void) {
    int N;
    int x, y, answer = 0;

    cin >> N;
    for (int i = 0; i < N; ++i)
    {
        cin >> x >> y;

        for (int j = y; j < y+10; ++j)
        {
            for (int k = x; k < x+10; ++k)
            {
                board[j][k] = 1;
            }
        }
    }

    for (int i = 0; i < 100; ++i)
    {
        for (int j = 0; j < 100; ++j)
        {
            if (board[i][j]) answer++;
        }
    }

    cout << answer << '\n';
    return 0;
}